### PR TITLE
Add OGV grant script

### PR DESCRIPTION
**Pull request by tomlinton**

### CODE REVIEW

File: token-transfer-server/src/scripts/insert_ogv_airdrop_grants.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+const BigNumber = require('bignumber.js')
+const Logger = require('logplease')
+const moment = require('moment')
+
+const db = require('../models')
+
+const { getBalance } = require('../lib/balance')
+
+Logger.setLogLevel(process.env.LOG_LEVEL || 'INFO')
+const logger = Logger.create('enqueue', { showTimestamp: false })
```
Comment (franckc) [inline token-transfer-server/src/scripts/insert_ogv_airdrop_grants.js]: supernit: rename the logger to reflect the script name or equivalent. and use logger throughout the script vs a mix of logger and console.log.

**Review by franckc — APPROVED** [review]
LGTM to add a grant for user's vested OGN.
And for unvested OGN I assume we'll use a different script or run a SQL query to create the associated OGV grants?

File: token-transfer-server/src/scripts/insert_ogv_airdrop_grants.js (hunk 1 above)
Comment (tomlinton) [reply token-transfer-server/src/scripts/insert_ogv_airdrop_grants.js]: Oops 👍

**Comment by tomlinton** [comment]
For investors there is no unvested OGN, and for everyone else I think they'll be manual inserts as the parameters aren't known.

**Comment by franckc** [comment]
> For investors there is no unvested OGN, and for everyone else I think they'll be manual inserts as the parameters aren't known.

Got it. Yes, makes sense...

**Pull request merged**